patient-appointment-management/README.md

```diff
@@ -0,0 +1,61 @@
+# patient-appointment-management
+
+Patient Appointment Management including notification and reminders sent via SMS
+
+## Pre-requisites
+
+- AWS Account with credentials to an admin-level IAM user
+  for deploying AWS resources via CloudFormation (aka deployer)
+
+### Environment variables
+
+This project requires some environment variables to be set. A file named `.env` is used to store the values for those environment variables. To keep your tokens and secrets secure, make sure to not commit the `.env` file in git. When setting up the project with `twilio serverless:init ...` the Twilio CLI will create a `.gitignore` file that excludes `.env` from the version history.
+
+In your `.env` file, set the following values:
+
+| Variable | Description | Required |
```

Comment:
Since the `.env` file for this app is fairly complex, it's very important this part of the `README` has documentation for what everything does.

Reply:
Check. fully documented README.md in the works.